entrypoint.sh

```diff
@@ -30,15 +33,21 @@ then
     # Switch to branch from current Workflow run
     git checkout $INPUT_BRANCH
 
-    if [ -z ${INPUT_FILE_PATTERN+x} ];
-    then
-        git add .
+    if is_defined "${INPUT_FILE_PATTERN}"; then
```

Comment:
In my tests the workflow fails, if no `file_pattern` is defined.
Did this work on your end?

Here's the error message:
```
/entrypoint.sh: line 36: INPUT_FILE_PATTERN: parameter not set
```

Reply:
Sorry for not reporting about the testing's status yesterday, I could've at least left some notes to inform you but I was exhausted. As you pointed out, following the `is_defined` approach still results in the same `param not set` error. However, it seems I was totally mistaken about [`args`](https://help.github.com/en/github/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#args) because, as per the docs, its a mandatory field if `entrypoint.sh` needs access to the input params.

> An array of strings that define the inputs for a Docker container. Inputs can include hardcoded strings. GitHub passes the args to the container's ENTRYPOINT when the container starts up.

So we need to add the `args` entry to `action.yml` accordingly:

```yml
runs:
   using: 'docker'
   image: 'Dockerfile'
   args:
      - ${{ inputs.commit_message }}
      - ${{ inputs.commit_options }}
      - ${{ inputs.branch }}
      - ${{ inputs.file_pattern }}
```
Also see 3889fd6. Even though this eliminates the `param not set` error, the container seemingly still doesn't respect default values(they remain empty values based on the tests/logs).

Reply:
Ah wait, there's an extra `s` in the metadata filename. The doc stresses the importance of naming the file `action.yml`.

> The metadata filename must be action.yml.

So the default value(s) issue presumably stems from not following their naming restrictions.

Reply:
Thanks for all the work and research!
And sorry, if you felt under pressure. Was not my intention!

Seems that during the migration, I've missed some critical steps or they weren't documented back then.
Will re-read the docs.

Reply:
Everything's fine, it's always fun to learn something new each day even if it's strenuous. I'll make sure whether it ultimately fixes the previous errors/unexpected behaviors and leave some final notes after.

Reply:
Had some issues with the Bash syntax but now it's solved. As empty strings are not allowed in commands, we have to avoid including `$INPUT_COMMIT_OPTIONS` if it's not defined(see 908446c). The solution comes from [here](https://unix.stackexchange.com/a/415992).

Could you also make sure all the existing & new features work appropriately?

Reply:
Ping @stefanzweifel. :)

Reply:
Will test your updates tonight.

Reply:
Forgot I could change files through GitHub's UI. Just tested it in my little test app and everything works fine.

@gomorizsolt I would merge and tag a new version tonight, if this is alright for you?

Reply:
Sure, go ahead.